validator/src/main/resources/validations/eks-cw-container-insight.yml

```diff
@@ -0,0 +1,4 @@
+-
+  validationType: "container-insight-eks-logs"
```

Comment:
let's change this name as well. Maybe `eks-fargate-container-insights-logs`.

Reply:
This determines the validator that is used. A new validator wasn't created, so this shouldn't be changed.

Reply:
@hossain-rayhan we won't be able to change this without making a new validator case; however making a new validator case may be repetitive since the container-insight-eks-logs validator case exists already

Reply:
Got it. Let's keep them for now.